lib/kotkowo_web/components/pagination_bar.ex

```diff
@@ -0,0 +1,74 @@
+defmodule KotkowoWeb.Components.PaginationBar do
+  @moduledoc false
+
+  use Phoenix.Component
+
+  attr :selected_page, :integer, required: true
+  attr :last_page, :integer, required: true
+
+  def pagination_bar(assigns) do
+    assigns =
+      assigns
+      |> assign(:previous_page, assigns.selected_page - 1)
+      |> assign(:next_page, assigns.selected_page + 1)
+
+    ~H"""
+    <div class="flex flex-row text-xl gap-x-4 text-lg">
+      <button :if={@selected_page != 1} phx-click="select_page" class="p-4" value={@selected_page - 1}>
```

Comment:
You're assuming that 1 is the starting index.
Such assumptions should be put in a module attribute:

```elixir
defmodule KotkowoWeb.Components.PaginationBar do
  @moduledoc false

  use Phoenix.Component

  @first_page 1
```

Reply:
and then refer to them instead of using magic numbers
https://en.wikipedia.org/wiki/Magic_number_(programming)